manager/model/user.go

```diff
@@ -0,0 +1,20 @@
+package model
+
+const (
+	UserStateEnabled  = "enable"
+	UserStateDisabled = "disable"
+)
+
+type User struct {
+	Model
+	Email             string `gorm:"column:email;size:256;uniqueIndex;not null" json:"email"`
+	Name              string `gorm:"column:name;size:256;uniqueIndex;not null" json:"name"`
+	EncryptedPassword string `gorm:"column:encrypted_password;size:1024" json:"encrypted_password"`
+	Avatar            string `gorm:"column:avatar;size:256;not null" json:"avatar"`
+	Phone             string `gorm:"column:phone;size:256;not null" json:"phone"`
+	PasswordSalt      string `gorm:"column:password;size:256;not null" json:"password"`
```

Comment:
Remove `PasswordSalt`.
Refer to https://medium.com/@jcox250/password-hash-salt-using-golang-b041dc94cb72